### DISCUSSION TITLE
Zabbix Agent on the postgresql STS?

### DISCUSSION THREAD

**Discussion by nrlpenneo** [zabbix-community/helm-zabbix#111] [General]
Is there a way to deploy the agent on the postgresql sts, in the same way there is an agent next to the server, so we can monitor the DB instance itself, specifically the mount point where the data is. I've run out twice, and I couldn't find a way to monitor this. The data is stored in a pvc, but could not find a way to monitor the free space on that PVC/mount point.

**Comment by aeciopires** [comment]
Hello @nrlpenneo!

Sorry for delay in my answer.

It`s a good idea. What do you think @fibbs?

**Comment by nrlpenneo** [comment]
Any progress on this?
